docs/metastores.md

```diff
@@ -0,0 +1,98 @@
+
+> Note: some database systems such as Postgres don't create databases automatically. Database admins need to create them manually before running Polaris server.
+```angular2html
+<persistence-unit name="polaris" transaction-type="RESOURCE_LOCAL">
+    <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+    <class>org.apache.polaris.core.persistence.models.ModelEntity</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityActive</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityChangeTracking</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityDropped</class>
+    <class>org.apache.polaris.core.persistence.models.ModelGrantRecord</class>
+    <class>org.apache.polaris.core.persistence.models.ModelPrincipalSecrets</class>
+    <class>org.apache.polaris.core.persistence.models.ModelSequenceId</class>
+    <shared-cache-mode>NONE</shared-cache-mode>
+    <properties>
+      <property name="jakarta.persistence.jdbc.url"
+        value="jdbc:h2:file:tmp/polaris_test/filedb_{realm}"/>
+      <property name="jakarta.persistence.jdbc.user" value="sa"/>
+      <property name="jakarta.persistence.jdbc.password" value=""/>
+      <property name="jakarta.persistence.schema-generation.database.action" value="create"/>
+    </properties>
+  </persistence-unit>
+```
+
+Build with H2 dependency and run Polaris service:
+> polaris> ./gradlew --no-daemon --info -PeclipseLink=true -PeclipseLinkDeps=com.h2database:h2:2.3.232 clean shadowJar
+> polaris> java -jar  polaris-service/build/libs/polaris-service-*.jar server ./polaris-server.yml
+
+### Postgres
+
+The following shows a sample configuration for Postgres database.
+
+```angular2html
+  <persistence-unit name="polaris" transaction-type="RESOURCE_LOCAL">
+  <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+  <class>org.apache.polaris.core.persistence.models.ModelEntity</class>
+  <class>org.apache.polaris.core.persistence.models.ModelEntityActive</class>
+  <class>org.apache.polaris.core.persistence.models.ModelEntityChangeTracking</class>
+  <class>org.apache.polaris.core.persistence.models.ModelEntityDropped</class>
+  <class>org.apache.polaris.core.persistence.models.ModelGrantRecord</class>
+  <class>org.apache.polaris.core.persistence.models.ModelPrincipalSecrets</class>
+  <class>org.apache.polaris.core.persistence.models.ModelSequenceId</class>
+  <shared-cache-mode>NONE</shared-cache-mode>
+  <properties>
+    <property name="jakarta.persistence.jdbc.url"
+              value="jdbc:postgresql://localhost:5432/{realm}"/>
```

Comment:
So from my local testing, I noticed I can set this to any database name and it will still work (using a simple realm in this case which is the default realm). Do we want to add example on how to use multi-realms?

Reply:
Yes, I was looking for that earlier but didn't find much. Haven't tried out the multi realms setup as of now.